src/router.js

```diff
@@ -24,6 +25,11 @@ export default new Router({
       name: 'notices',
       component: Notices
     },
+    {
+      path: '/event/add',
+      name: 'addEvent',
+      component: addEvent
```

Comment:
See if we should rename the component to AddEvent

Reply:
Renamed.